src/consts.ts

```diff
@@ -19,10 +19,40 @@ const is = (...x) => 音韻地位.屬於(...x);
 /** @type { 音韻地位["判斷"] } */
 const when = (...x) => 音韻地位.判斷(...x);
 
-if (!音韻地位) return [
-  // 在此輸入方案選項……
+if (!音韻地位) return [  // 在此輸入方案選項；以下提供一些範本，方案設定格式：https://github.com/nk2028/tshet-uinh-deriver-tools/wiki/%E6%8E%A8%E5%B0%8E%E6%96%B9%E6%A1%88%E8%A8%AD%E5%AE%9A%E9%A0%85%E5%88%97%E8%A1%A8%E6%A0%BC%E5%BC%8F
+  //['選項代號（非必要）|選項名\n新行描述（非必要）\n可以加入更多行', [2, '選項1', '選項2']],
+  //['是非題，是非皆可作預設選項', false],
```

Comment:
```suggestion
  //['是非選項\n預設可為是（true）或非（false）', false],
```

「是非題」「皆可作」果然還是聽着怪怪的🤔這樣會不會更好點

Reply:
```suggestion
  [`核取方塊選項
    選項説明
    預設值須為勾選（true）或不勾選（false）之一`, false],
```

Reply:
不然：
```suggestion
  [`核取方塊選項
    預設值須為勾選（true）或不勾選（false）之一`, false],
```